ddtrace/internal/writer.py

```diff
@@ -60,40 +58,142 @@ def __init__(
         shutdown_timeout=DEFAULT_TIMEOUT,
         sampler=None,
         priority_sampler=None,
+        buffer_size=20 * 1000000,
+        max_payload_size=5 * 1000000,
```

Comment:
you can bump this probably, previous default in the agent is 10, latest default is 50

Reply:
i think the problem is how up to date people's agents are 😆

Reply:
oh yeah, but we can probably bump to 8 or 9 (maybe even 10 if we truly don't ever go over)

Reply:
yep! I bumped it up